packages/p2p/src/pages/my-ads/my-ads.jsx

```diff
@@ -18,6 +19,10 @@ const MyAdsState = ({ message }) => (
 
 const MyAds = () => {
     const { general_store, my_ads_store, my_profile_store } = useStores();
+    const is_poi_poa_verified =
+        general_store.poi_status === identity_status_codes.VERIFIED &&
+        general_store.p2p_poa_required &&
+        general_store.poa_status === document_status_codes.VERIFIED;
```

Comment:
ive seen this in another page, could we not create a computed value for this or a hook for this to return if it is poi or poa verified?

Reply:
This will be replaced by a hook which is common across the app.